### ISSUE TITLE
Quotes needed in windows makefile

### ISSUE THREAD

**Issue by alexander-novo** [openssl/openssl#1085]
I'm trying to build openssl on Windows 10 x64 and I was running into the problem of nmake failing because my openssl was in a directory with a space in the name.

I had this little bit in my makefile:

```
configdata.pm: C:\C++ Libs\openssl\Configurations\windows-makefile.tmpl $(SRCDIR)\Configure
	@echo "Detected changed: $?"
	@echo "Reconfiguring..."
	$(PERL) $(SRCDIR)\Configure reconf
	@echo "**************************************************"
	@echo "***                                            ***"
	@echo "***   Please run the same make command again   ***"
	@echo "***                                            ***"
	@echo "**************************************************"
	@exit 1
```

And changing it to this fixed the problem:

```
configdata.pm: "C:\C++ Libs\openssl\Configurations\windows-makefile.tmpl" $(SRCDIR)\Configure
	@echo "Detected changed: $?"
	@echo "Reconfiguring..."
	$(PERL) $(SRCDIR)\Configure reconf
	@echo "**************************************************"
	@echo "***                                            ***"
	@echo "***   Please run the same make command again   ***"
	@echo "***                                            ***"
	@echo "**************************************************"
	@exit 1
```

**Comment by levitte** [comment]
Doesn't it need quotes around `$(SRCDIR)\Configure`?

**Comment by alexander-novo** [comment]
All I can tell you is that the second result got me to #1086.

**Comment by levitte** [comment]
You might want to have a look at #1125

**Comment by mattcaswell** [comment]
Ping @levitte, can this be closed now #1125 is merged?

**Comment by levitte** [comment]
Yes
